### ISSUE TITLE
Non-reproducible results from command line [BUG]

### ISSUE THREAD

**Issue by HernandoMV** [MouseLand/cellpose#843]
**Describe the bug**
I am having different results with the GUI and the CLI, presumably with same arguments

**To Reproduce**
1. Run cyto model on the attached image on the GUI with diameter 120, and the rest are default parameters. I get 2 ROIs
2. Run CLI as in the log below. I get 0 ROIs.

**Run log**
python -m cellpose --dir /mnt/d/temp/test/ --pretrained_model cyto --diameter 120 --use_gpu --chan 0 --verbose
2024-01-11 15:26:17,559 [INFO] WRITING LOG OUTPUT TO /home/hernando/.cellpose/run.log
2024-01-11 15:26:17,559 [INFO]
cellpose version:       2.2.3
platform:               linux
python version:         3.10.10
torch version:          2.0.1+cu117
2024-01-11 15:26:18,094 [INFO] ** TORCH CUDA version installed and working. **
2024-01-11 15:26:18,094 [INFO] >>>> using GPU
2024-01-11 15:26:18,108 [INFO] >>>> running cellpose on 1 images using chan_to_seg GRAY and chan (opt) NONE
2024-01-11 15:26:18,109 [INFO] ** TORCH CUDA version installed and working. **
2024-01-11 15:26:18,109 [INFO] >>>> using GPU
2024-01-11 15:26:18,109 [INFO] >> cyto << model set to be used
2024-01-11 15:26:18,199 [INFO] >>>> model diam_mean =  30.000 (ROIs rescaled to this size during training)
2024-01-11 15:26:18,200 [INFO] >>>> using diameter 120.000 for all images
2024-01-11 15:26:18,201 [INFO] 0%|          | 0/1 [00:00<?, ?it/s]
2024-01-11 15:26:18,208 [INFO] ~~~ FINDING MASKS ~~~
2024-01-11 15:26:19,037 [INFO] >>>> TOTAL TIME 0.83 sec
2024-01-11 15:26:19,050 [INFO] 100%|##########| 1/1 [00:00<00:00,  1.18it/s]
2024-01-11 15:26:19,050 [INFO] 100%|##########| 1/1 [00:00<00:00,  1.18it/s]
2024-01-11 15:26:19,050 [INFO] >>>> completed in 0.956 sec

**Screenshots**
![image](https://github.com/MouseLand/cellpose/assets/6049065/2397790f-f750-4647-8f9d-bb3a704ecf0b)

**Actual image**
![MJFF_Cali_011_C05_010_Time_Z_3_enhanced-scaled](https://github.com/MouseLand/cellpose/assets/6049065/24d55a2b-f92c-4a4b-8921-2f47af13ec0f)

**Comment by carsen-stringer** [comment]
hmm are you sure it is absolutely the same image? or maybe the GUI is normalizing it different, what dtype is the image?

**Comment by HernandoMV** [comment]
Yeah it is the same image. A .tif file

**Comment by carsen-stringer** [comment]
are you still having this issue? going to close this one and keep track in #972
